spec.md

```diff
@@ -390,20 +388,28 @@ Example error response (invalid `id` provided):
 
 #### Collection HTTP API
 
-#### Collection Javascript Client API
-
 ## Resources and Blobs
 
 ### Blob Data Model
 
+A logical unit of data, in transit or at rest.
```

Comment:
I'm not sure the 'logical' part here is important, and I found myself being like... 'is it though'.

I think 'a unit of data' is accurate and more clear.

(Sometimes I refer to a resource/object as a 'logical' thing because really all it is is a name that points to different 'concrete' data representations over time. And that doesn't really apply to a blob of specific binary data imho)

Reply:
```suggestion
A unit of data, in transit or at rest.
```

Reply:
@gobengo good point, the `logical` part is not that useful there. Removed.